packages/core/test/integration/child-proxy/Echo.ts

```diff
@@ -26,9 +26,6 @@ export class Echo {
 
   public exit(code: number) {
     process.exit(code);
-    return new Promise(() => {
```

Comment:
This is a test case, it is actually supposed to never resolve.

Reply:
It is not even running tho :/ process.exit works like return so that promise line is not even executed. I mean as long as its fine with you its ok (tslint or sth just kept removing it :x)

Reply:
Ah, I see. You are correct in removing it then.